src/views/Settings.vue

```diff
@@ -54,70 +55,96 @@
                   <Icon name="external-link" class="ml-1" />
                 </UiButton>
               </a>
-              <UiButton class="text-left width-full mb-2 d-flex px-3">
-                <div class="text-gray mr-2">{{ $t('settings.name') }}</div>
-                <input v-model="form.name" class="input flex-auto" required />
-              </UiButton>
-              <UiButton class="text-left width-full mb-2 d-flex px-3">
-                <div class="text-gray mr-2">{{ $t('settings.about') }}</div>
-                <input v-model="form.about" class="input flex-auto" />
-              </UiButton>
-              <UiButton
+              <UiInput
+                v-model="form.name"
+                :warning="inputErrors.includes('name')"
+              >
+                <template v-slot:label> {{ $t(`settings.name`) }} * </template>
```

Comment:
The * sign should appear without a space, like this
```suggestion
                <template v-slot:label>{{ $t(`settings.name`) }}*</template>
```